queens.hs

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
I think the removal of the first line was a great contribution.

Reply:
Agree! You are a role-model when it comes to not having redundant newlines!